docs/blog/posts/buckets_as_conceptual_hydr.md

```diff
@@ -0,0 +1,103 @@
+
+A simple differential equation for the ale level in Sir Edmond's bucket is:
+
+$$
+\frac{dh}{dt} = -k \sqrt{h}
+$$
+
+Where h(t) is the ale at time \( t \), k is a proportionality constant that governs the rate of outflow. Its solution through seperation of variables is:
+
+$$
+h(t) = \left(\sqrt{h_0} - \frac{k}{2}t\right)^2
+$$
+
+Where h_0 is the initial ale level in the bucket at time t = 0. This gives us the opportunity to track volumes of ale through this bucket, and match the fluxes from buckets with data collected on real-world hydrological systesm. This is, in a nutshell, the field of computational hydrology, we just need to dress up and add complications to this bucket, and off we go.  
+
+<!-- more -->
+
+## A walk through the watershed
+
+Going back again to undergraduate, I was building my first watershed model (Frame, 2010). I spend weeks staring at a computer screen, writing code, processing data, and thinking through discretization of Sir Edmond's ale bucket. I went out to do some field work for a different project in the Carmel River Watershed, a valley I've been to a dozen times. But after spending so much time thinking about this watershed digitally, well really just a digital bucket representing the watershed, a sense of dred came over me as I looked down the Carmel Valley (Figured below). I was realizing how much my precious computer model was missing, seeing the errors in my assumptions, the variety of vegetation in areas my models represents as homogeneous, the property boundaries with drainage features. Nothing in the analog system seemed to match my digitization. There is overwhelming complexity in every watershed, river reach and hillslope, if our eyes are open.  
+
+<p align="center">
+  <img src="pics/jmframe_dampoint.jpg" alt="jmframe in the field, Carmel River Watershed, 2010" width="300"/>
+</p>
+
+But in hydrology, we consider representing a delayed flow of water as a bucket a type of physical conceptualization. And as a matter of fact, this goes back a long way to Instantaneous Unit Hydrograph (IUH). In my MS level hydrology class, we learned that the IUH from linear reservoirs turns into a beta distribution (Nash 1960). This is a type of theoretical physical understanding of hydrology through buckets.  
+
+The equation for the Instantaneous Unit Hydrograph (IUH) from linear reservoirs is:  
+
+$$
+q(t) = \frac{t^{n-1} e^{-t/k}}{k^n \Gamma(n)}
+$$
+
+Where q(t) is the flow rate at time \( t \), k is the storage coefficient (related to the delay in each reservoir), n is the number of reservoirs (controls the shape of the hydrograph), Gamma(n) is the gamma function.  
```

Comment:
Suggest using math symbols for all variables e.g., $q$

Reply:
I made the change in the latest commit

Reply:
Thanks. I finally figured out how to make it work. Added a markdown_extension:   - pymdownx.arithmatex:
      generic: true

Reply:
Thank you for adding math plugin!!